el8/Dockerfile

```diff
@@ -3,7 +3,8 @@ LABEL maintainer="CMS Build"
 LABEL name="CMS Worker Node on EL - Buildtime dependencies"
 
 # Dependencies on libidn libXmu libstdc++-static might be cleaned up in the future
-RUN dnf install --disablerepo=epel -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exporter \
+RUN rpm -qa --queryformat '%{NAME} %{VERSION} %{RELEASE}\n' >> /tmp/versions-grid.tmp &&\
```

Comment:
does `rpm -qa` return sorted list? If not then better to add `| sort` for each `rpm -qa`

Reply:
It is not sorted by default, so I will do `sort` for each `rpm -qa`